docs/config/README.md

```diff
@@ -2908,9 +2908,9 @@ By default the module will be shown if any of the following conditions are met:
 format = "via [⍱ $version](bold white) "
 ```
 
-## VLang
+## V
```

Comment:
Pretty sure this should still be VLang

Reply:
```suggestion
## VLang
```

Reply:
`[citation needed]`

Reply:
What? Do you need a reason for it? If so, look at Golang, it's Golang, not Go. This should be done for consistency. Or change both I suppose.

Reply:
The initial purpose of this PR is to change "Golang" to "Go". Please read the full diff.